benches/benches/bevy_ecs/change_detection.rs

```diff
@@ -41,7 +54,7 @@ impl BenchModify for Sparse {
     }
 }
 
-const RANGE_ENTITIES_TO_BENCH_COUNT: std::ops::Range<u32> = 5..7;
+const ENTITIES_TO_BENCH_COUNT: [u32; 2] = [5000, 50000];
```

Comment:
Definitely like showcasing a wider range, and doing it directly.

Nit: a vec might be nicer to refactor.